src/QuizAtom.tsx

```diff
@@ -1,7 +1,317 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { SvgCheckmark, SvgCross } from '@guardian/src-icons';
+import { css } from 'emotion';
+import { neutral } from '@guardian/src-foundations/palette';
+import { body } from '@guardian/src-foundations/typography';
 
-import { QuizAtomType } from './types';
+import {
+    QuizAtomType,
+    Question as QuestionType,
+    Answer as AnswerType,
+} from './types';
 
-export const QuizAtom = ({ id }: QuizAtomType): JSX.Element => (
-    <div data-atom-id={id}>QuizAtom</div>
+const enum AnswerState {
```

Comment:
I like this way of encapsulating the states 👍

Reply:
Haha thanks Jon, both you and Nic looked slightly horrified by how this was done before! It's much nicer now for sure.